controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -203,17 +213,17 @@ module.exports = async (req, res) => {
                             a.only_received_dm_from_user_following,
                             a.is_backdoor_user,
                             a.followers_count,
-                            a.karma_score,
-                            b.topic_id
+                            a.karma_score
                           FROM users a
                           INNER JOIN user_topics b ON a.user_id = b.user_id
                           WHERE 
                             a.is_anonymous = false
                             AND a.is_banned = false
                             AND a.blocked_by_admin = false
                             AND a.karma_score > 30
-                            AND a.user_id != :admin_user_id
                             AND b.topic_id NOT IN (:topics)
+                            AND a.user_id NOT IN (:allUserIds)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

When passing `allUserIds` to the SQL query, ensure that it's converted back to an array if you've implemented the previous suggestions to use a `Set`. This is because SQL queries do not support `Set` objects.
```diff
- AND a.user_id NOT IN (:allUserIds)
+ AND a.user_id NOT IN (Array.from(:allUserIds))
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->